modules/admin/proc-rbac-config-conditional-policy-file.adoc

```diff
@@ -0,0 +1,118 @@
+[id='proc-rbac-config-conditional-policy-file_{context}']
+= Configuring conditional policies defined in an external file
+
+You can configure and manage conditional policies that are defined in an external file.
+
+.Prerequisites
+* You are logged in to your {ocp-short} account using the {ocp-short} web console.
+* You have defined conditional policies in a YAML file, which includes role references, permission mappings, and conditions.
++
+--
+The following is an example of a YAML file defining conditional policies:
+
+[source,yaml]
+----
+---
+result: CONDITIONAL
+roleEntityRef: 'role:default/test'
+pluginId: catalog
+resourceType: catalog-entity
+permissionMapping:
+  - read
+  - update
+conditions:
+  rule: IS_ENTITY_OWNER
+  resourceType: catalog-entity
+  params:
+    claims:
+      - 'group:default/team-a'
+      - 'group:default/team-b'
+---
+result: CONDITIONAL
+roleEntityRef: 'role:default/test'
+pluginId: catalog
+resourceType: catalog-entity
+permissionMapping:
+  - delete
+conditions:
+  rule: IS_ENTITY_OWNER
+  resourceType: catalog-entity
+  params:
+    claims:
+      - 'group:default/team-a'
+----
+--
+* You have defined roles and associated policies in a CSV file that serves as a basis for creating roles and permissions.
++
+For more information, see xref:ref-rbac-conditional-policy-definition_title-authorization[].
+
+.Procedure
+
```

Comment:
At this point, the YAML and CSV file location is not determined:

- You have the files on your local environment? 
- You uploaded the files to OpenShift? (into which OpenShift object?)
- You uploaded the files to a remote URL that is accessible for OpenShift ?

From the first example, I assume you uploaded the files to OpenShift, but as a redar I would prefer to assert.